homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from typing import Any
+import logging
+
+from homeassistant.components.fan import FanEntity, FanEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+FULL_SUPPORT = (
+    FanEntityFeature.SET_SPEED
+)
+
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the Lutron fans."""
+    devs = []
+    for area_name, device in hass.data[LUTRON_DEVICES]["fan"]:
+        dev = LutronFan(area_name, device, hass.data[LUTRON_CONTROLLER],hass, device.uuid, device.name, FULL_SUPPORT, None)

```

Comment:
Not sure why linting checks haven't thrown a flag for this yet, but lines should be 79 characters.

Reply:
I cleaned this up and will get it pushed up